CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_snapchat_ads v0.9.0
+[PR #34](https://github.com/fivetran/dbt_snapchat_ads/pull/34) introduces the following updates:
+
+## Schema Updates
+
+**2 total changes • 1 breaking change**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| `snapchat_ads__campaign_country_report` | New Transforms Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
+| `snapchat_ads__campaign_region_report` | New Transforms Model |   |   |  Uses `campaign_geo_region_daily_report` source table  |
+
+Please note that these are disabled by default. To enable them, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+vars:
+    snapchat__using_campaign_country_report: true # Necessary for the snapchat_ads__campaign_country_report model. False by default. Requires the campaign_geo_country_daily_report source table
+    snapchat__using_campaign_region_report: true # Necessary for the snapchat_ads__campaign_region_report model. False by default. Requires the campaign_geo_region_daily_report source table
+```
+
```

Comment:
Could you add an entry for the new passthrough column variables?

Reply:
updated